Dockerfile

```diff
@@ -1,12 +1,56 @@
-FROM jpetazzo/dind
+FROM jpetazzo/dind AS builder
 
 RUN cd /opt && \
```

Comment:
The base image already has Java... no need to install it here. Move this to line 50, after `FROM jpetazzo/dind`

Reply:
updated